src/basic/time-util.h

```diff
@@ -30,6 +30,7 @@ typedef struct triple_timestamp {
 } triple_timestamp;
 
 typedef enum TimestampStyle {
+        TIMESTAMP_EPOCH,
```

Comment:
bikeshedding: wouldn't TIMESTAMP_UNIX be more appropriate? That's how it's usually called?

Reply:
I guess that makes sense, updated - naming stuff is hard :(